src/pages/memos-dtd/[functionId]/form/MemosDtdForm.js

```diff
@@ -0,0 +1,121 @@
+import { FormControlLabel, Grid, Checkbox } from '@mui/material'
+import { useContext, useEffect, useState } from 'react'
+import * as yup from 'yup'
+import FormShell from 'src/components/Shared/FormShell'
+import toast from 'react-hot-toast'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { useInvalidate } from 'src/hooks/resource'
+import { ResourceIds } from 'src/resources/ResourceIds'
+
+import ResourceComboBox from 'src/components/Shared/ResourceComboBox'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { useForm } from 'src/hooks/form'
+import { FinancialRepository } from 'src/repositories/FinancialRepository'
+
+import { SystemRepository } from 'src/repositories/SystemRepository'
+
+export default function MemosDtdForm({ labels, maxAccess, dtId, functionId }) {
+  const { getRequest, postRequest } = useContext(RequestsContext)
+
+  const invalidate = useInvalidate({
+    endpointId: FinancialRepository.FIDocTypeDefaults.qry
+  })
+
+  const { formik } = useForm({
+    initialValues: {
+      dtId: dtId,
+      plantId: '',
+      recordId: dtId
+    },
+    maxAccess,
+    enableReinitialize: true,
+    validateOnChange: true,
+    validationSchema: yup.object({
+      dtId: yup.string().required(' ')
+    }),
+    onSubmit: async obj => {
+      const response = await postRequest({
```

Comment:
response not used anywhere remove it and call API directly
await postRequest({
        extension: FinancialRepository.FIDocTypeDefaults.set,
        record: JSON.stringify(obj)
      })